perf/src/sigverify.rs

```diff
@@ -608,12 +609,18 @@ pub fn shrink_batches(batches: &mut Vec<PacketBatch>) {
     batches.truncate(last_valid_batch);
 }
 
-pub fn ed25519_verify_cpu(batches: &mut [PacketBatch], reject_non_vote: bool, packet_count: usize) {
+pub fn ed25519_verify_cpu(
+    batches: &mut [PacketBatch],
+    reject_non_vote: bool,
+    packet_count: usize,
+    active_threads_hist: Option<&mut histogram::Histogram>,
```

Comment:
Why is this an `Option`?

Reply:
There are some cases where we don't care to spin up a Stats structure and collect this information. For example, unit testing, benches, verifying votes int he vote listener (at least currently)